src/dbt_score/evaluation.py

```diff
@@ -67,6 +70,7 @@ def evaluate(self) -> None:
             self._formatter.model_evaluated(
                 model, self.results[model], self.scores[model]
             )
+            self.min_model_score = min(self.min_model_score, self.scores[model].value)
```

Comment:
I'm not sure we should calculate the minimum here, I think there'd be more value giving a list of all models which fall below the minimum allowed instead?

Reply:
How would something like this be ?

```🥉 customers_view (score: 7.3)
    WARN (medium) dbt_score.rules.generic.columns_have_description: Columns lack a description: c_name, c_comment.
    OK   dbt_score.rules.generic.has_description
    WARN (medium) dbt_score.rules.generic.has_owner: Model lacks an owner.
    OK   dbt_score.rules.generic.public_model_has_example_sql
    OK   dbt_score.rules.generic.sql_has_reasonable_number_of_lines

🥈 customer_nationalities (score: 9.3)
    OK   dbt_score.rules.generic.columns_have_description
    OK   dbt_score.rules.generic.has_description
    OK   dbt_score.rules.generic.has_owner
    WARN (low) dbt_score.rules.generic.public_model_has_example_sql: The model description does not include an example SQL query.
    OK   dbt_score.rules.generic.sql_has_reasonable_number_of_lines

Project score: 8.3 🥈
2024-06-17 17:55:00,214 ERROR [dbt_score.cli] Model model.view_test.customers_view scored 7.3 which is less than `fail_any_model_under` setting of 10.0
2024-06-17 17:55:00,214 ERROR [dbt_score.cli] Model model.view_test.customer_nationalities scored 9.3 which is less than `fail_any_model_under` setting of 10.0
```

Reply:
Not sure I would use the logger, the output should match what's above to be efficiently parsed by humans.
I suggest to simplify it a bit:
```
Error: fail_any_model_under = 10.0
Model model.view_test.customers_view scored 7.3
Model model.view_test.customer_nationalities scored 9.3
```
and
```
Error: fail_project_under = 10.0
```

What do you think?